modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -371,6 +371,32 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
 };
 /** @} */
 
+namespace gapi {
+namespace streaming {
+/**
+ * @brief Ask G-API to set specific queue capacity for streaming execution.
```

Comment:
Not "ask G-API to set" But just "Specify"

Reply:
Done